src/constants.js

```diff
@@ -8,7 +8,7 @@ export const RIGHT_ENROLMENTOFFICER = 121501;
 export const RIGHT_CLAIMADMINISTRATOR = 121601;
 export const RIGHT_PAYERS = 121801;
 export const RIGHT_LOCATIONS = 121901;
-export const RIGHT_TASK_EXECUTIONER_GROUPS = 190000;
+export const RIGHT_TASK_EXECUTIONER_GROUPS = 190001;
```

Comment:
Just to confirm - do we have a ticket to move task executioner views from the administrator view to the separate module? Backend is separate and so should be this. It is a contribution to the Admin menu, but not integral part of it. If there's a ticket let me know. if it will be addressed later on in development we can merge this PR and add change afterwards.

Reply:
I check and I don't see a ticket for that. I will create it.